docs/controls/access-controls/AC-02(05).md

```diff
@@ -0,0 +1,16 @@
+# Access Control  
+## AC-02(05) - ACCOUNT MANAGEMENT | INACTIVITY LOGOUT
+
+The organization requires that users log out when [Assignment: organization-defined time-period of expected inactivity or description of when to log out].  
+
+AC-2 (5) Additional FedRAMP Requirements and Guidance:  
+Guidance: Should use a shorter timeframe than AC-12  
+
+### TDP Implementation  
+TDRS is configured to log out users out after 30 minutes of inactivity.  
+
+If there has been 20 minutes of inactivity, TDRS will alert the user that the session will time out soon and give them the option to either extend the session or logout of the system. If the user chooses to extend the session, TDRS will refresh the session. If the user elects to log out of the system it will end the session and log the user out of the system.  
```

Comment:
```suggestion
If there has been 20 minutes of inactivity, TDP will alert the user that the session will time out soon and give them the option to either extend the session or logout of the system. If the user chooses to extend the session, TDP will refresh the session. If the user elects to log out of the system it will end the session and log the user out of the system.  
```